tla-io/src/test/scala/at/forsyte/apalache/io/quint/TestQuintEx.scala

```diff
@@ -122,6 +122,9 @@ class TestQuintEx extends AnyFunSuite {
 
     // Names and parameters
     val name = e(QuintName(uid, "n"), QuintIntT())
+    val nameBoolSet = e(QuintName(uid, "Bool"), QuintSetT(QuintBoolT()))
+    val nameIntSet = e(QuintName(uid, "Int"), QuintSetT(QuintIntT()))
+    val nameNatSet = e(QuintName(uid, "Nat"), QuintSetT(QuintIntT()))
```

Comment:
If you don't plan to reuse the named setr, you could create then inline in the test, just FYI. But this also looks great.

Reply:
Oh, just found `nam`.

Reply:
1c06d9d70